Behat/Context/DrupalExtendedContext.php

```diff
@@ -497,4 +497,38 @@ public function userWithMailNotExists($mail) {
      $this->userWithMailExists($mail, FALSE);
    }
 
+  /**
+   * Checks that text appears before another text.
+   *
+   * Example:
+   * I should see "Element 1" text before "Element 2" from "li" element.
```

Comment:
«I should see "Element 1" text before "Element 2" from "li" element.» -> «I should see "Element 1" text before "Element 2" in the "content" region»